server/config.go

```diff
@@ -0,0 +1,75 @@
+package server
+
+import (
+	"bytes"
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+)
+
+// Server implements the RuntimeService and ImageService
+type Config struct {
+	// Root is a path to the "root directory" where all information not
+	// explicitly handled by other options will be stored.
+	Root string `toml:"server.root"`
```

Comment:
I would remove `server` - we won't going to have a real client, I would call it just `ocid`? or move everything to the top level indent?